lib/src/push.dart

```diff
@@ -62,7 +62,7 @@ class Push {
   }
 
   void startTimeout() {
-    if (_timeoutTimer == null) return;
+    if (_timeoutTimer != null) return;
 
```

Comment:
@phamhieu I feel like I could've found this sooner, but this seemed to be causing the two `unsubscribe` tests to fail. Here is the implementation of `realtime-js` https://github.com/supabase/realtime-js/blob/master/src/lib/push.ts#L66

Now all tests are passing 🎉